src/main/java/uk/gov/hmcts/reform/notificationservice/model/servicebus/in/NotificationMsg.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.notificationservice.model.servicebus.in;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class NotificationMsg {
+
+    public final String zipFileName;
+    public final String jurisdiction;
+    public final String poBox;
+    public final String documentControlNumber;
+    public final ErrorCode errorCode;
```

Comment:
do we really need error code?

Reply:
are values defined by provider?
making enum cause coupling to consumers.

Reply:
the specs were written by us and provider implemented this way